asdf.nu

```diff
@@ -74,6 +74,30 @@ module asdf {
         }
     }
 
+    def "complete asdf plugin versions all" [context: string] {
+        let plugin = $context | str trim | split words | last
+        ^asdf list all $plugin 
+        | lines 
+        | each { |line| $line | str trim } 
+        | prepend "latest"
+    }
+
+    def "complete asdf plugin versions installed" [context: string] {
+        let plugin = $context | str trim | split words | last
+        let versions = ^asdf list $plugin 
+        | lines 
+        | each { |line| $line | str trim }
+        | each { |version| if ($version | str starts-with "*") {{value: ($version | str substring 1..), description: "current global"}} else {{value: $version, description: ""}} }
```

Comment:
nitpick: The `*` star indicates the version is installed, not that it is 'current global'. I'm not familiar with nushell so I may not be reading this correctly.